### PR TITLE
Update/popup for stuck bridge token

### PR DESCRIPTION
**Pull request by trungbach**
add func move stuck token in oraibridge to oraichain

### CODE REVIEW

File: src/pages/BalanceNew/StuckOraib/useGetOraiBridgeBalances.tsx [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
+    ORAI_BRIDGE_CHAIN_ID, ORAI_BRIDGE_RPC
 } from 'config/constants';
 import { toDisplay } from 'libs/utils';
 import { useEffect, useState } from 'react';
-import token from "reducer/token";
-import axios from 'rest/request';
-import { Balance, BalancesOraiBridge } from '../type';
 
 export type RemainingOraibTokenItem = TokenItemType & { amount: string };
 const ORAIB_DECIMALS = 18;
```
Comment (ducphamle2) [inline src/pages/BalanceNew/StuckOraib/useGetOraiBridgeBalances.tsx:10]: this part should be edited. ORAIB_DECIMALS is not always 18. You can get the actual decimals based on the token denom filtered from the list in bridgeTokens.ts file

File: src/pages/BalanceNew/StuckOraib/index.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            {loading ?
+                <img src={loadingGif} alt='loading-gif' width={30} height={30} />
+                :
+                <div className={styles.stuckText} onClick={handleMove}>Move Stuck Token</div>
+            }
+            <TooltipIcon
+                placement="bottom-end"
+                content={remainingOraib.map(token => {
+                    const { Icon } = token
+                    const oraibDecimals = 18;
```
Comment (ducphamle2) [inline src/pages/BalanceNew/StuckOraib/index.tsx:30]: This also needs to be edited. A rule of thumb is that you should not hard code any value. If you are planning to hard code anything, its best to ask others first. We should get the oraib decimals from filtering the token info from the list in bridgeTokens.ts file

File: src/pages/BalanceNew/StuckOraib/index.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                    return (
+                        <div className={styles.stuckToken}>
+                            <div>
+                                <Icon width={20} height={20} className={styles.icon} />
+                                <span className={styles.name}>{token.name}</span>
+                            </div>
+                            <TokenBalance
+                                balance={{
+                                    amount: token.amount ?? '0',
```
Comment (ducphamle2) [inline src/pages/BalanceNew/StuckOraib/index.tsx:39]: token.amount should likely be put in a toDisplay function here

File: src/pages/BalanceNew/index.tsx [hunk 4]
```diff
@@ -718,12 +711,44 @@ const Balance: React.FC<BalanceProps> = () => {
 
   const navigate = useNavigate();
 
+  // Move oraib2oraichain
+  const [moveOraib2OraiLoading, setMoveOraib2OraiLoading] = useState(false);
+  const { remainingOraib } = useGetOraiBridgeBalances(moveOraib2OraiLoading);
+
+  const moveOraibToOraichain = async () => {
```
Comment (ducphamle2) [inline src/pages/BalanceNew/index.tsx:718]: This function should be put outside of the tsx file to write test cases. It should return the list of from, to & transfer amount. Then, we have another function that calls moveOraibToOraichain, then probably call await transferIbcCustom

File: src/pages/BalanceNew/StuckOraib/useGetOraiBridgeBalances.tsx [hunk 5]
```diff
@@ -21,14 +22,18 @@ export default function useGetOraiBridgeBalances(moveOraib2OraiLoading: boolean)
             }
             const client = await StargateClient.connect(ORAI_BRIDGE_RPC);
             const data: readonly Coin[] = await client.getAllBalances(oraiBridgeAddress);
-            const remainingOraib = data.filter((item: Coin) => toDisplay(item.amount, ORAIB_DECIMALS) > 0)
-                .map((item: Coin) => {
```
Comment (ducphamle2) [inline src/pages/BalanceNew/StuckOraib/useGetOraiBridgeBalances.tsx]: I think this file can have several test cases to test filtering & querying OraiBridge balances

**Review by ducphamle2 — APPROVED** [review]
lgtm

**Review by haunv3 — COMMENTED** [review]
lgtm

**Pull request merged**